### DISCUSSION TITLE
"Bettle" - Wordle style webpage with this data

### DISCUSSION THREAD

**Discussion by matthoffner** [kyleskom/NBA-Machine-Learning-Sports-Betting#186] [Show and tell]
Thanks for this project! I've been using the script to update a Google Sheet in "Wordle" style pick-em UI:

https://bettle.netlify.app/

**Comment by deperto** [comment]
> Thanks for this project! I've been using the script to update a Google Sheet in "Wordle" style pick-em UI:
> 
> 
> 
> https://bettle.netlify.app/

This is great 😊🙌

**Comment by ykk9595** [comment]
> Thanks for this project! I've been using the script to update a Google Sheet in "Wordle" style pick-em UI:
> 
> https://bettle.netlify.app/

How can i use this

**Reply by matthoffner** [reply]
It's a bit of a work in progress, but the odds should show up and your previous pick is saved in local storage to show you how did in theory.  

I'm just running the script manually in a Colab notebook with some code to write to the sheet. Still working on automating some of this. The spread pick is my own formula based on net expected value from the model.

The Google sheet is a slightly modified example from the Odds API examples that runs every day. I have another cron job there that checks the winners from the NBA api and some formulas setup to evaluate.
